ui/cypress/e2e/collectors.test.ts

```diff
@@ -122,5 +122,54 @@ describe('Collectors', () => {
 
       cy.getByTestID('setup-instructions').should('not.exist')
     })
+
+    it('can filter telegraf configs correctly', () => {
```

Comment:
🔥 🙌 🔥 
THANKS FOR TESTS

Reply:
love it! what do y'all think about throwing in a comment linking to the issue? it might be cool to link specific tests to bugfixes?

Reply:
Great idea!